src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -316,6 +328,7 @@ HRESULT STDMETHODCALLTYPE CorProfiler::AssemblyLoadFinished(AssemblyID assembly_
 
 HRESULT STDMETHODCALLTYPE CorProfiler::ModuleLoadFinished(ModuleID module_id,
                                                           HRESULT hr_status) {
+  Info("ModuleLoadFinished: ", module_id);
```

Comment:
Left over from testing?

Reply:
ups!

Reply:
fixed in https://github.com/DataDog/dd-trace-dotnet/pull/1007/commits/9e7bcb3955022de47db4e80e29c33db9f64b69c2